pywinauto/win32structures.py

```diff
@@ -119,21 +118,21 @@ def _reduce(self):
 
 #LPTTTOOLINFOW = POINTER(tagTOOLINFOW)
 #PTOOLINFOW = POINTER(tagTOOLINFOW)
-BOOL = c_int
-BYTE = c_ubyte
+BOOL = wintypes.BOOL
+BYTE = wintypes.BYTE
 CHAR = c_char
-DWORD = c_ulong
-HANDLE = c_void_p
-HBITMAP = c_long
-LONG = c_long
-LPVOID = c_void_p
+DWORD = wintypes.DWORD

```

Comment:
I would suggest using `wintypes` directly in `win32functions` etc. So these definitions in `win32structures` would be redundant. I can agree with current change as preliminary step before we totally get rid of this re-definition.